### PR TITLE
Update restore-backup.md

### PR DESCRIPTION
**Pull request by NielsKunst**
Added section on restore failure

### CODE REVIEW

**Review by stasfilin — APPROVED** [review]
LGTM

File: content/en/docs/developerportal/operate/backups/restore-backup.md [hunk 1]
```diff
@@ -91,15 +91,25 @@ For a Free App, you can only restore available backups which are made from the a
 
 3. Click **Yes** to confirm the restoration.
 
-## 5 Format of a Backup File{#format-of-backup-file}
+## 5 Restoring After a Backup Fails {#restore-after-fail}
+
+When a backup restore fails, you will be notified via your app's **Backup Activity** log (found in the **Backups** page of the *Developer Portal*). When this happens, all data that was restored until the point of failure will be present in your database. This will leave the database only partially restored; not all data from the backup file will be present in your database. 
+
+Your database must be large enough to hold the decompressed size of the database as stored in your backup file's [db folder](#db-folder), plus an overhead of 2.25 GB. This overhead is employed during the restoration process. For example, if the size of your decompressed database in the **db folder** is 6 GB, your database must have at least 8.25 GB of storage.
```
Comment (ConnorLand) [inline content/en/docs/developerportal/operate/backups/restore-backup.md]: @nelis37 I tried to add an example here to showcase the math. Are these terms and figures appropriate? If they need to be tweaked to be more realistic, please do so based on your experience:

"For example, if the size of your decompressed database in the **db folder** is 6 GB, your database must have at least 8.25 GB of storage."

Alternatively, if you think this example isn't helpful I can just delete it.

Comment (NielsKunst) [reply content/en/docs/developerportal/operate/backups/restore-backup.md]: That makes sense. We can make it more explicit by including a database size that we actually have in our CLoud Recourse packs, I suggest:

For example, if you run your app in a S21 Cloud Recourse Pack your database size is 10 GB. To be able to restore a backup the size of your decompressed database in the db folder must not be bigger than 7.75 GB, to allow for the overhead of 2.25 GB.

And maybe add a link to the Cloud Resource Pack page on those words but I am not sure how to do that here. 

https://docs.mendix.com/developerportal/deploy/mendix-cloud-deploy/#resource-pack

Comment (ConnorLand) [reply content/en/docs/developerportal/operate/backups/restore-backup.md]: I replaced my example with your example, and linked the doc. Thanks very much for the link!

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**